scripts/build/unzip.ps1

```diff
@@ -1,14 +1,16 @@
 <#
     .SYNOPSIS
     This script wraps c#'s io.compression.zipfile to unzip a zip file.
-    The script is intended to be used within the VSO build to unzip the winobjc SDK.
+    The script is intended to be used within the VSO build to unzip a NuGet package.
 #>
 param(
-    #NOTE: using Parameter(Mandatory) will hang the VSO build job since powershell will block waiting for user input.
+    #NOTE: using Parameter(Mandatory) will hang the VSO build job since
+    #      powershell will block waiting for user input.
     [string]$zipFile = $(throw 'Mandatory parameter "zipFile" is not set.'),
     [string]$outputDir = $(throw 'Madatory parameter "outputDir" is not set.')
     )
 
+$zipFile = gci $zipFile
```

Comment:
Converts to absolute path (it was useful to have a * in the path)

---
In reply to: [103060152](https://github.com/Microsoft/WinObjC/pull/2097#discussion_r103060152) [](ancestors = 103060152)